scipy/spatial/transform/rotation.py

```diff
@@ -1460,3 +1462,54 @@ def __getitem__(self, indexer):
                [ 0.57735027,  0.57735027, -0.57735027,  0.        ]])
         """
         return self.__class__(self._quat[indexer], normalized=True)
+
+    @classmethod
+    def random(cls, num=None, random_state=None):
+        """Generate uniformly distributed rotations.
+
+        Parameters
+        ----------
+        num : None or int
+            Number of random rotations to generate. If None, then a single
```

Comment:
Alternative wording I like "if None (default), ...". But you surely can keep your variant if you prefer.